Detectors/EMCAL/reconstruction/include/EMCALReconstruction/CTFCoder.h

```diff
@@ -145,7 +156,8 @@ void CTFCoder::decode(const CTF::base& ec, VTRG& trigVec, VCELL& cellVec)
       cellVec.emplace_back(cell);
       cellCount++;
     }
-    trigVec.emplace_back(ir, firstEntry, entries[itrig]);
+    uint32_t trigBits = trigger[itrig];
```

Comment:
@mfasDa @mpoghos here you should implement the inverse conversion (in case the content of the `TriggerRecord::mTriggerBits` has different layout wrt the 16 bit word stored in the CTF.

Reply:
OK, here we can then do the inverse coding adding a setCodedTriggerBits() to o2::emcal::TriggerRecord.